web_app/utils/fucntools.py

```diff
@@ -15,3 +21,112 @@ def get_client_ip(request: Request) -> str:
         ip = request.client.host
 
     return ip
+
+
+def download_parquet_file(
+    bucket_name: str | None = GS_BUCKET_NAME, protocol_name: str = None
+) -> None:
+    """
+    Downloads parquet file to local storage from Google Cloud Storage
+    :param bucket_name: Google Cloud Storage bucket name
+    :param protocol_name: Protocol name
+    :return: None
+    """
+    assert protocol_name in [item.value for item in ProtocolIDCodeNames]
```

Comment:
Do not use `assert` in production code. 
assert can be disabled by os.
`assert` is allowing to use only in test cases.
Please, change it to `if` condition, if no -> raise custom exception.